src/test/rustdoc-json/reexport/glob_extern.rs

```diff
@@ -4,15 +4,20 @@
 #![feature(no_core)]
 
 // @is glob_extern.json "$.index[*][?(@.name=='mod1')].kind" \"module\"
-// @is glob_extern.json "$.index[*][?(@.name=='mod1')].inner.is_stripped" "true"
+// @is - "$.index[*][?(@.name=='mod1')].inner.is_stripped" "true"
```

Comment:
Sorry to nit-pick, but I wonder: if it's worth to use `-` instead of the whole filename? That just messes up the alignment and makes the code harder to read, IMHO.